akka-docs/src/main/paradox/additional/rolling-updates.md

```diff
@@ -0,0 +1,90 @@
+# Rolling Updates
+
+A rolling update is the process of replacing one version of the system with another without downtime.
+The changes can be new code, changed dependencies such as new Akka version, or modified configuration.
+
+In Akka, rolling updates are typically used for a stateful Akka Cluster where you can't run two separate clusters in
+parallel during the update, for example in blue green deployments.
+
+For rolling updates related to Akka dependency version upgrades and the migration guides, please see
```

Comment:
Might be worth an extra section below with some basic information similar to the other ones.

Reply:
I would but Patrik wanted to be sure we do not duplicate content here.